apps/target-app/app.py

```diff
@@ -63,68 +65,73 @@ def filtered_data() -> pd.DataFrame:
     return sample_df.reset_index(drop=True)
 
 
-with ui.navset_bar(id="tabs", title="Monitoring"):
-    with ui.nav_panel("Training Dashboard"):
-        with ui.layout_columns():
-            with ui.card():
-                ui.card_header("Model Metrics")
-
-                @render_plotly
-                def metric():
-                    df_value = df()
-                    df_filtered = df_value[df_value["account"] == input.account()]
-                    if input.metric() == "ROC Curve":
-                        return plot_auc_curve(
-                            df_filtered, "is_electronics", "training_score"
-                        )
-                    else:
-                        return plot_precision_recall_curve(
-                            df_filtered, "is_electronics", "training_score"
-                        )
-
-                ui.input_select(
-                    "metric",
-                    "Metric",
-                    choices=["ROC Curve", "Precision-Recall"],
-                )
-            with ui.card():
-                ui.card_header("Training Scores")
-
-                @render_plotly
-                def score_dist():
-                    df_value = df()
-                    df_filtered = df_value[df_value["account"] == input.account()]
-                    return plot_score_distribution(df_filtered)
-
-        with ui.card(full_screen=True):
-            with ui.card_header():
-                "Data"
-            with ui.popover(title="Download"):
+ui.nav_spacer()
+
+with ui.nav_panel("Training Dashboard"):
```

Comment:
You can use `ui.nav_panel()` at the top-level in an Express app to automatigically have the page be a `page_navbar()`. Express finds the `ui.sidebar()` and the `ui.nav_panel()` and automatically collects those into `ui.page_navbar()`.